onnxmltools/convert/lightgbm/_parse.py

```diff
@@ -90,3 +123,7 @@ def parse_lightgbm(model, initial_types=None, target_opset=None,
         raw_model_container.add_output(variable)
 
     return topology
+
+
+def default_initial_types(model):
+    return [('features', FloatTensorType(shape=(1, model.n_features_)))]
```

Comment:
It should be return [('features', FloatTensorType(shape=(**None**, model.n_features_)))] as the number of rows is usually unknown. We recently introduced that change in onnxconverter-common.

Reply:
I have changed the code, but it seems the test does not handle it very well. Is it expected?

Reply:
Well, I have retracted the automatic input inference part. It might need changes in code outside the scope of this PR. It would be better to make a separate PR for that.